Model/Order.php

```diff
@@ -201,11 +201,14 @@ public function escapeJsQuote($data, $quote = '\'')
 
     /**
      * @param $data
-     * @return string
+     * @return array|string|null
      */
     public function escapeReturn($data)
     {
-        return trim(str_replace(["\r\n", "\r", "\n"], ' ', $data));
+        if (!is_null($data) && !is_array($data)) {
```

Comment:
@GaryHaddon ... why not 

return is_string($data) ? trim(str_replace(["\r\n", "\r", "\n"], ' ', $data)) : $data;

Reply:
@srenon either works.